src/gl/vglRenderer.js

```diff
@@ -90,6 +90,17 @@ var vglRenderer = function (arg) {
     canvas.attr('class', 'webgl-canvas');
     canvas.css('display', 'block');
     $(m_this.layer().node().get(0)).append(canvas);
+
+    if (window.contextPreserveDrawingBuffer) {
```

Comment:
The screenshots are actually the canvas drawing buffers, not actual screenshots.  There are virtues to this approach: it is easy to ignore the window border, tabs, etc. and it doesn't matter if the div is aligned the same way.  It ignores non-canvas data, which is both good and bad.  It does require that you can ask the canvas for the drawing buffer, though, hence this flag,